src/App.js

```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import InputCalculator from './components/Calculator';
 
 function App() {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/158527668-28513421-9d25-4c16-9819-551d36c408eb.png)

However, I'm getting this (just a tiny button):

![image](https://user-images.githubusercontent.com/57421823/158527756-b6edfa3b-aa29-4bc7-90dd-1480fcbba983.png)

This seems to be the source of the rendering problem:

![image](https://user-images.githubusercontent.com/57421823/158527819-ebddf7d5-3a19-454a-82b5-d569144d45db.png)

Therefore:

- [ ] Kindly make sure the calculator renders correctly and matches the given design.

That way, we will move closer to approval 💪